src/pages/tag/[tag].astro

```diff
@@ -0,0 +1,76 @@
+---
+import Layout from '@/layouts/DefaultLayout.astro'
+import BlogPostEntry from '@/components/BlogPostEntry.vue'
+import { resolvePath, uniq } from '@/utils/helpers.js'
+import { sortedPosts } from '@/utils/sort'
+
+export async function getStaticPaths () {
+  const allPosts = sortedPosts(await Astro.glob('../../posts/*.{md,mdx}'))
+  const uniqueTags = uniq(
+    allPosts.map((post: any) => post.frontmatter?.tags || []).flat()
+  )
```

Comment:
extracts all `tags` from the existing blog-posts and create `/tag/string` pages for each.